tests/lib/hardware/libvirt.py

```diff
@@ -50,16 +50,15 @@
 
 class Node(NodeBase):
     def __init__(self, name, role, tags, conn, image_path,
-                 network, disk_number, memory, ssh_public_key, ssh_private_key,
-                 workspace):
+                 network, disk_number, memory, workspace):
         super().__init__(name, role, tags)
         self._conn = conn
         self._image_path = image_path
         self._network = network
         self._disk_number = disk_number,
         self._memory = memory * 1024 * 1024
-        self._ssh_public_key = ssh_public_key
-        self._ssh_private_key = ssh_private_key
+        self._ssh_public_key = workspace.public_key
```

Comment:
we could just drop theses vars and use workspace directly. but can be done in a follow-up cleanup